src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaMetadataController.java

```diff
@@ -0,0 +1,30 @@
+package uk.ac.ebi.atlas.hcalandingpage;
+
+import com.google.gson.JsonObject;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static uk.ac.ebi.atlas.utils.GsonProvider.GSON;
+
+@RestController
+public class HcaMetadataController {
+    private final static String SPECIES = "Homo sapiens";
+    private final HcaMetadataService hcaMetadataService;
+
+    public HcaMetadataController(HcaMetadataService hcaMetadataService) {
+        this.hcaMetadataService = hcaMetadataService;
+    }
+
+    @GetMapping(value = "/json/metadata/hca",
+            produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public String getHCAMetadata() {
```

Comment:
Look, this is a sad camel:
![image](https://user-images.githubusercontent.com/4425744/76077044-e54dc100-5f97-11ea-9fcb-dac5590ada73.png)